### PR TITLE
Bring feedback form closer to compliance w/ current expected look & feel

### LINKED ISSUE #2497: Update layout of Feedback form
As part of the effort to standardize our feedback forms, below are recommendations for updated the SW feedback form. Note that the basic layout updates have been made to Earthworks (https://github.com/sul-dlss/earthworks/pull/691) although this might only be partially relevant since I believe SW is still on BL 6.

Here's the [current SW feedback form](https://searchworks.stanford.edu/view/4797413), as seen by non-authenticated users:

Screen Shot 2020-07-15 at 12 10 50 PM

---

The primary updates needed are:

- [ ] **A:** Add a Bootstrap alert-primary component to the top of the form and move the "Reporting from: current page URL" string to that component. 

- [ ] **B:** Move the "Check system status" link into the alert-primary component. It should be right-aligned, or given ~40px left margin from the "Reporting from:" string. The check status link should have `font-weight: 600;` to draw more attention to it than the neighboring "Reporting from: link. 

- [ ] **C:** When the "Report wrong cover image" button is present, it should be displayed below the alert-primary component, left-aligned with the form fields.

Ideally the horizontal sizing of the form fields should be updated to more closely match what was done in [Earthworks](https://github.com/sul-dlss/earthworks/pull/691). Moving the Check status and other elements from the side of the form fields to above them should provide the extra width needed. We're trying to avoid this narrow field situation:

Screen Shot 2020-07-15 at 12 12 15 PM

### Authenticated user version
If the current user is authenticated, we show an additional element -- the Chat link -- and remove the Captcha. When both are present, I think the Chat link should come before the "Report wrong cover image" button.

Screen Shot 2020-07-15 at 12 09 54 PM

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #2497

This PR makes visual changes to the feedback form and the connection form in
SearchWorks. Also includes:

* Fix test
* Increase contrast of info/success/warning/danger states (including alerts)
  * This brings Bootstrap 5 colors, which provide better accessibility, into Bootstrap 3
* Remove an unused CSS class

# Screenshot

Screenshot from 2021-05-11 11-48-58

What do you think, @ggeisler ? Anything I missed, or any feedback to offer?

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@ggeisler here you go:

Screenshot from 2021-05-11 14-17-17

File: app/helpers/feedback_form_helper.rb [hunk 1]
```diff
@@ -1,11 +1,8 @@
 module FeedbackFormHelper
   def render_feedback_form(form_type)
-    case form_type
-    when 'connection'
-      render 'shared/feedback_forms/connection_form'
-    else
-      render 'shared/feedback_forms/form'
```
Comment (mjgiarlo) [inline app/helpers/feedback_form_helper.rb:7]: Why have two identical forms when you can have one?

**Comment by ggeisler** [comment]
Sorry, @mjgiarlo, one little thing if you don't mind making one more update.

I think the `.librarian-chat: padding-bottom: 25px;` in `app/assets/stylesheets/modules/feedback-form.scss` is likely a holdover from when that link was presented to the side of the form fields rather than above. As a result, there's too much whitespace between the chat/cover image elements and the top of the form. Can you make that `.librarian-chat: padding-bottom: 12px;` instead? I think that should make it look closer to my mockup in  https://github.com/sul-dlss/SearchWorks/issues/2497 in terms of the whitespace above and below those elements.

**Comment by mjgiarlo** [comment]
@ggeisler :speech_balloon: 
> Sorry, @mjgiarlo, one little thing if you don't mind making one more update.
> 
> I think the `.librarian-chat: padding-bottom: 25px;` in `app/assets/stylesheets/modules/feedback-form.scss` is likely a holdover from when that link was presented to the side of the form fields rather than above. As a result, there's too much whitespace between the chat/cover image elements and the top of the form. Can you make that `.librarian-chat: padding-bottom: 12px;` instead? I think that should make it look closer to my mockup in #2497 in terms of the whitespace above and below those elements.

Ah, yes, even nicer now. Thanks, done!

File: spec/features/quick_report_spec.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@
     click_link 'Feedback'
     expect(page).to have_css('button.btn-quick-report')
     click_button 'Report wrong cover image'
-    expect(page).to have_css('div.alert-success', text: 'Thank you! Your feedback has been sent.')
+    expect(page).to have_css('div.alert-info', text: 'Thank you! Your feedback has been sent.')
```
[the remark this answers is not in the document]
Comment (mjgiarlo) [reply spec/features/quick_report_spec.rb]: Indeed, it was fine, and I don't know why I changed it. Probably to get a test to pass in an early WIP state when I had things screwy? Anyway, over it now!

[the remark this answers is not in the document]
Comment (mjgiarlo) [reply spec/features/quick_report_spec.rb]: Resolving.

**Review by cbeer — APPROVED** [review]

**Pull request merged**